contracts/ScheduleUtils.sol

```diff
@@ -132,11 +122,10 @@ library ScheduleUtils {
       rampStartTime = schedule.cliffs[schedule.cliffCount - 1].time;
     }
     uint256 rampTotalTime = schedule.endTime - rampStartTime;
-    int256 rampTimeElapsed = int256(atTime) - int256(rampStartTime);
-    return
-      rampTimeElapsed <= 0
-        ? 0
-        : (rampTotalAmount * uint256(rampTimeElapsed)) / rampTotalTime;
+    if (atTime < rampStartTime) {
```

Comment:
was removing the ternary recommended in favor of multi-return?

Reply:
not explicitly no, it was just about not using signed ints.  I'll factor tat back out.